### DISCUSSION TITLE
Welcome to react-native-multi-tv-app-sample Discussions!

### DISCUSSION THREAD

**Discussion by giolaq** [AmazonAppDev/react-native-multi-tv-app-sample#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ksvslk** [comment]
Hello, thank you for launching this initiative.

I’m curious about the support for older versions of webOS and Tizen. Are there any known limitations for projects like this, particularly related to Chromium or potential JavaScript restrictions?

**Reply by giolaq** [reply]
Thank you @ksvslk ! Currently the multi tv app can be built in plain js/html ( with the web build ). Then with some manual steps you should be able to build it and run across different web-based TVs ( I've tested on web os for example ).
Feel free to start a conversation https://github.com/AmazonAppDev/react-native-multi-tv-app-sample/discussions/new?category=ideas . I'm happy to follow up also with the steps to build the app for Tizen/LG WebOS
